OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -64,7 +64,7 @@ public void StartGame(Arguments args)
 			var connect = args != null ? args.GetValue("Launch.Connect", null) : null;
 			if (!string.IsNullOrEmpty(connect))
 			{
-				var parts = connect.Split(':');
+				var parts = connect.Replace("openra://", "").Split(':');
```

Comment:
Well, it is just that I now need to write scripts whose interpreting shells will differ across distributions. Ironically your call will make this more platform-specific and error prone so you are not only making this harder, but also a lot more fragile. I will make some experiments. Let's see how this turns out.